src/sdkRuntimeModels.ts

```diff
@@ -150,7 +156,7 @@ export interface SDKProduct {
     Position?: number;
     CouponCode?: string;
     TotalAmount?: number;
-    Attributes?: { [key: string]: string };
+    Attributes?: Record<string, unknown> | undefined;
```

Comment:
`creatProduct` technically assigns this to `null` if there are no attributes. Probably should be a future refactor to have it be `undefined`.

```suggestion
    Attributes?: Record<string, unknown> | null;
```